src/Range/Generic/Range`1.cs

```diff
@@ -18,5 +18,25 @@ public Range(IRange<T> value)
 
         public bool IsMinInclusive { get; set; }
         public bool IsMaxInclusive { get; set; }
+
+        public static implicit operator string(Range<T> range)
+        {
+            return range.ToString();
+        }
+
+        public static implicit operator Range<T>(string range)
```

Comment:
Maybe this should be `explicit` since `Range.FromString<T>` will throw an exception if `range` is not properly formatted and of type `T`.

And while I really like

```csharp
Range<int> range = "[1,2]";
```

I'm okay with:

```csharp
Range<int> range = (Range<int>)"[1,2]";
```

considering most will do:

```csharp
var range = (Range<int>)"[1,2]";
```

Ref. https://msdn.microsoft.com/en-us/library/xhbhezf4.aspx